website/docs/user-group/user/invitations.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Invitations
+description: "Learn how to create an invitation URL for new users to enroll."
+---
+
+Invitations are another way to create a user, by inviting someone to join your authentik instance, as a new user.
+
+With invitations, you can either email an enrollment invitation URL to one or more specific recipients with pre-defined credentials, or you can email a URL to users, who can then log in and define their own credentials.
+
+:::info
+
+You can also create a policy to see if the invitation was ever used.
+
+:::
+
+## Create an invitation
+
+Invitations are yet another way to connect Flows, Stages, and Prompts in order to create a speficic user task or worflow.
```

Comment:
s/speficic/specific/

"Mike, this [encyclopedia is so old](https://mst3k.fandom.com/wiki/MST3K_820_-_Space_Mutiny#:~:text=Opening%3A%20Crow%20and%20Tom%20think%20the%20encyclopedias%20that%20Mike%20has%20are%20a%20little%20dated.) Congress is spelled with an 'f'! What is this, congreffs"?